invenio_app_ils/holidays/api.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""ILS holidays API."""
```

Comment:
I am not 100% sure in the end, but we went for `library closures` naming?
```suggestion
"""ILS location closures API."""
```